rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -42,7 +41,7 @@ class ROCALCOCOIterator(object):
            Epoch size.
     """
 
-    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False, num_anchors=120087):
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False, num_anchors=8732):
```

Comment:
Can you please define constants NUM_ANCHORS_COCO (so it is more readable for user). Also please define usecase for retinanet as well

Reply:
The num_anchors are not used in this example so it can be removed. A SSD Retinanet example will be added in the SSD training PR

Reply:
The num_anchors are not used in this example so its removed from the arguments